[2024-06-05T16:51:46.747+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-05T16:51:46.792+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: s3_postgres.fetch_recent_data scheduled__2024-06-04T00:00:00+00:00 [queued]>
[2024-06-05T16:51:46.813+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: s3_postgres.fetch_recent_data scheduled__2024-06-04T00:00:00+00:00 [queued]>
[2024-06-05T16:51:46.815+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 4
[2024-06-05T16:51:46.839+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_recent_data> on 2024-06-04 00:00:00+00:00
[2024-06-05T16:51:46.858+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=82) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-05T16:51:46.863+0000] {standard_task_runner.py:63} INFO - Started process 84 to run task
[2024-06-05T16:51:46.862+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 's3_postgres', 'fetch_recent_data', 'scheduled__2024-06-04T00:00:00+00:00', '--job-id', '725', '--raw', '--subdir', 'DAGS_FOLDER/pipe.py', '--cfg-path', '/tmp/tmp0l4iav9n']
[2024-06-05T16:51:46.865+0000] {standard_task_runner.py:91} INFO - Job 725: Subtask fetch_recent_data
[2024-06-05T16:51:46.955+0000] {task_command.py:426} INFO - Running <TaskInstance: s3_postgres.fetch_recent_data scheduled__2024-06-04T00:00:00+00:00 [running]> on host 6dbcf9923072
[2024-06-05T16:51:47.062+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='s3_postgres' AIRFLOW_CTX_TASK_ID='fetch_recent_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-04T00:00:00+00:00'
[2024-06-05T16:51:47.064+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-05T16:51:47.076+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2024-06-05T16:51:47.115+0000] {logging_mixin.py:188} INFO - Data fetch complete
[2024-06-05T16:51:47.116+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-05T16:51:47.118+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-05T16:51:47.131+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=s3_postgres, task_id=fetch_recent_data, run_id=scheduled__2024-06-04T00:00:00+00:00, execution_date=20240604T000000, start_date=20240605T165146, end_date=20240605T165147
[2024-06-05T16:51:47.160+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-05T16:51:47.180+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-05T16:51:47.182+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-05T17:47:35.668+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-05T17:47:35.692+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: s3_postgres.fetch_recent_data scheduled__2024-06-04T00:00:00+00:00 [queued]>
[2024-06-05T17:47:35.701+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: s3_postgres.fetch_recent_data scheduled__2024-06-04T00:00:00+00:00 [queued]>
[2024-06-05T17:47:35.702+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 4
[2024-06-05T17:47:35.716+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_recent_data> on 2024-06-04 00:00:00+00:00
[2024-06-05T17:47:35.723+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=136) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-05T17:47:35.725+0000] {standard_task_runner.py:63} INFO - Started process 138 to run task
[2024-06-05T17:47:35.725+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 's3_postgres', 'fetch_recent_data', 'scheduled__2024-06-04T00:00:00+00:00', '--job-id', '807', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpnhmfm9gn']
[2024-06-05T17:47:35.726+0000] {standard_task_runner.py:91} INFO - Job 807: Subtask fetch_recent_data
[2024-06-05T17:47:35.780+0000] {task_command.py:426} INFO - Running <TaskInstance: s3_postgres.fetch_recent_data scheduled__2024-06-04T00:00:00+00:00 [running]> on host b9a8b9be1324
[2024-06-05T17:47:35.891+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='s3_postgres' AIRFLOW_CTX_TASK_ID='fetch_recent_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-04T00:00:00+00:00'
[2024-06-05T17:47:35.893+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-05T17:47:35.908+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2024-06-05T17:47:35.962+0000] {logging_mixin.py:188} INFO - Data fetch complete
[2024-06-05T17:47:35.963+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-05T17:47:35.964+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-05T17:47:35.977+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=s3_postgres, task_id=fetch_recent_data, run_id=scheduled__2024-06-04T00:00:00+00:00, execution_date=20240604T000000, start_date=20240605T174735, end_date=20240605T174735
[2024-06-05T17:47:36.020+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-05T17:47:36.051+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-05T17:47:36.054+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
